memoirs-of-the-godly/modernized/01-memoirs-of-the-godly.adoc

```diff
@@ -1250,12 +1281,11 @@ the men`'s meeting disowned him from membership, and his heretical opinions,
 as ever having been any part of the creed of the Society,
 as appears by the following document, transcribed from the minutes of that meeting.
 
-For the clearing of the precious truth of God, professed by us,
-his people called Quakers,
-from the occasion of stumbling and reproach given by Jeffery Bullock`'s pernicious doctrine,
-in affirming that he neither expects justification
-nor condemnation by that Christ that died,
-or was put to death, at Jerusalem.
+[.embedded-content-document]
+--
+
+[.blurb]
+=== For the clearing of the precious truth of God, professed by us, his people called Quakers, from the occasion of stumbling and reproach given by Jeffery Bullock`'s pernicious doctrine, in affirming that he neither expects justification nor condemnation by that Christ that died, or was put to death, at Jerusalem.
```

Comment:
I thought this looked a little weird being so long, but I wasn't sure what to do with it. Maybe it's fine!